### PR TITLE
Resolve Issue #279

### PR DESCRIPTION
**Pull request by stevedsimkins**
Added "Check your Network" section starting line 102, making sure users have their network set to Rinkeby

### CODE REVIEW

File: NFT_Game/en/Section_3/Lesson_4_Checking_For_Character_NFT.md [hunk 1]
```diff
@@ -99,6 +99,24 @@ Import ethers into your `App.js` file:
...
+const checkNetwork = async () => {
+  try { 
+    if(window.ethereum.networkVersion !== '4'){
+      alert("Please connect to Rinkeby!")
+    }
+  } catch(error){
+    console.log(error)
+  }
+}
+```
```
Comment (adilanchian) [inline NFT_Game/en/Section_3/Lesson_4_Checking_For_Character_NFT.md:116]: Can you add a bit of formatting here like so - 

```javascript
const checkNetwork = async () => {
  try { 
    if (window.ethereum.networkVersion !== '4') {
      alert("Please connect to Rinkeby!")
    }
  } catch(error) {
    console.log(error)
  }
}
```

File: NFT_Game/en/Section_3/Lesson_4_Checking_For_Character_NFT.md [hunk 2]
```diff
@@ -99,6 +99,24 @@ Import ethers into your `App.js` file:
...
+
+Now anytime the page is loaded on the mainnet you will get a notice for your users to switch to rinkeby!
```
Comment (adilanchian) [inline NFT_Game/en/Section_3/Lesson_4_Checking_For_Character_NFT.md:118]: I think right before this, a small explanation of what the 4 value means here would be ideal!

**Review by adilanchian — CHANGES_REQUESTED** [review]
I love this - thanks for adding!

Just left a few comments for ya 🤘

File: NFT_Game/en/Section_3/Lesson_4_Checking_For_Character_NFT.md (hunk 1 above)
Comment (stevedsimkins) [reply NFT_Game/en/Section_3/Lesson_4_Checking_For_Character_NFT.md]: Added formatting to latest commit!

File: NFT_Game/en/Section_3/Lesson_4_Checking_For_Character_NFT.md (hunk 2 above)
Comment (stevedsimkins) [reply NFT_Game/en/Section_3/Lesson_4_Checking_For_Character_NFT.md]: Added explanation to latest comitt!

**Comment by stevedsimkins** [comment]
Updated the patch with the changes requesed!

**Comment by adilanchian** [comment]
@stevedsimkins this is DOPE. Thank you so much for this!!

**Comment by adilanchian** [comment]
@all-contributors please add @stevedsimkins for doc

**Comment by adilanchian** [comment]
ALSO - CONGRATS ON YOUR FIRST CONTRIBUTION

🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄

**Comment by stevedsimkins** [comment]
Thanks man!! Happy to help :)

**Pull request merged**